### PR TITLE
#179 replace the clockpicker with time select component in time proposition

### PR DESCRIPTION
**Pull request by rezk2ll**
**DEMO**:

- in a normal event ( end and begin time ): https://streamable.com/wd4oqv
- in an all-day event: https://streamable.com/wpgzcm

### CODE REVIEW

**Review by piavgh — DISMISSED** [review]

File: src/esn.calendar.libs/app/components/event-date-suggestion/event-date-suggestion.pug [hunk 1]
```diff
@@ -7,24 +7,30 @@
...
+              locale="{{ ::ctrl.locale }}",
+              time-format="{{ ::ctrl.timeFormat }}"
+            )
       .col-xs-12.col-md-4
         .form-group.nopadding
           .fg-line
             .checkbox.allday
               label
-                input(type="checkbox", ng-model='ctrl.allDay', ng-change="ctrl.setEventDates();")
+                input(type="checkbox", ng-model='ctrl.full24HoursDay', ng-change="ctrl.setEventDates();")
```
Comment (huy-ta) [inline src/esn.calendar.libs/app/components/event-date-suggestion/event-date-suggestion.pug:33]: This fix isn't related to this issue, right?

Comment (rezk2ll) [reply src/esn.calendar.libs/app/components/event-date-suggestion/event-date-suggestion.pug:33]: well to some extent it is, the time select does not behave correctly when using `allDay` ( which doesn't exist in the controller ).

the logic was not changed, I used `full24HoursDay` to judge whether the user wants to suggest another date ( the all-day tick is checked ) if not the time select appears.

Comment (huy-ta) [reply src/esn.calendar.libs/app/components/event-date-suggestion/event-date-suggestion.pug:33]: You should have a separate commit for this then.

Comment (rezk2ll) [reply src/esn.calendar.libs/app/components/event-date-suggestion/event-date-suggestion.pug:33]: alright

**Review by huy-ta — APPROVED** [review]

**Review by MichaelBailly — APPROVED** [review]

**Review by huy-ta — APPROVED** [review]

**Review by asmaNode — APPROVED** [review]

**Pull request merged**